src/bindings/python/tests_compatibility/runtime.py

```diff
@@ -13,6 +13,7 @@
 from ngraph.impl import Function, Node, PartialShape, Type
 from ngraph.opset1.ops import result
 from ngraph.utils.types import NumericData, get_shape, get_dtype
+from _pyngraph.util import get_ie_output_name
```

Comment:
@jiwaszki is it ok to import from `_pyngraph` directly here? I don't want to expose this new helper `get_ie_output_name` to the legacy API, just use it internally in our own test runner.

Reply:
If this is only for tests usage and you do not want to expose it for end users via init files, I think `_pyngraph` is absolutely fine.